src/nvim/getchar.c

```diff
@@ -144,7 +144,7 @@ static int KeyNoremap = 0;                  /* remapping flags */
 static char_u typebuf_init[TYPELEN_INIT];       /* initial typebuf.tb_buf */
 static char_u noremapbuf_init[TYPELEN_INIT];    /* initial typebuf.tb_noremap */
 
-static int last_recorded_len = 0;       /* number of last recorded chars */
+static unsigned int last_recorded_len = 0;      // number of last recorded chars
```

Comment:
I was going to do that, but I wasn't sure if I should change the size since last_recorded_len doesn't get that big. However, you are right it does make more sense to have it as size_t.